.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
can we delete this file? also maybe we add .vscode to .gitignore?

Reply:
yes i'll add that